artichoke-backend/src/extn/core/time/mruby.rs

```diff
@@ -19,7 +19,7 @@ pub fn init(interp: &mut Artichoke) -> InitializeResult<()> {
         .value_is_rust_object()
         // Constructor
         .add_self_method("now", time_self_now, sys::mrb_args_none())?
-        .add_self_method("at", time_self_at, sys::mrb_args_req(1))?
+        .add_self_method("at", time_self_at, sys::mrb_args_req(1) | sys::mrb_args_opt(3))?
```

Comment:
there's another sys args helper that does this operation: https://artichoke.github.io/artichoke/artichoke_backend/sys/fn.mrb_args_req_and_opt.html

```suggestion
        .add_self_method("at", time_self_at, sys:: mrb_args_req_and_opt(1, 3))?
```